styles.css

```diff
@@ -85,16 +92,24 @@ main{
     border-radius: 4px;
 }
 
-#lp{
+.numbers:hover{
+    background-color: #E8E8E8;
+    border: 2px solid #363636;
+    color: #363636;
+    cursor: pointer;
+}
+
+
+#leftparenthesis{
     grid-area: 2/1/2/1;
 }
-#rp{
+#rightparenthesis{
```

Comment:
Use camelCase for `id`

Reply:
But, it’s nice you have updated the names :)

Reply:
ty